aws-sessionstore-dynamodb.gemspec

```diff
@@ -15,7 +15,8 @@ Gem::Specification.new do |spec|
   spec.test_files    = spec.files.grep(%r{^(test|spec|features)/})
   spec.require_paths = ["lib"]
 
-  # Require 1.85.0 for user_agent_frameworks config
```

Comment:
Please keep this comment - useful for SDK feature tracking.

Reply:
Sorry, I inadvertently erased it. I put it back.